docs/faq/general/which-version-control-systems-do-you-support.md

```diff
@@ -10,9 +10,9 @@ On Codacy Cloud we support the following Git providers:
 
 Besides these, on [Codacy self-hosted](https://www.codacy.com/self-hosted) we also support:
 
--   GitHub Enterprise
--   GitLab Enterprise
--   Bitbucket Server
+-   GitHub Enterprise (version 2.20.3 or later)
+-   GitLab Enterprise (version 12.6.2-ee or later)
+-   Bitbucket Server (version 6.6.0 or later)
```

Comment:
Which Git provider versions should we list here?

These are the versions of our Git provider instances for the dev environment, where I assume that we test the Codacy features against. But this doesn't necessarily mean that previous versions of the Git providers won't work with Codacy Self-hosted as well. :thinking:

Reply:
For example, to address the issue [CY-3302](https://codacy.atlassian.net/browse/CY-3302?focusedCommentId=44559), we should list the minimum version **11.6** for GitLab Enterprise.

Reply:
We may want to make some specific effort on some specific version, and install that version instead, but I think that we should suggest the minimum as the version we have installed and test. So this would make sense for me for now